Gemfile.lock

```diff
@@ -928,7 +928,7 @@ DEPENDENCIES
   pry-rails (~> 0.3)
   puma (~> 5.3.2)
   pundit (~> 2.1)
-  pundit-matchers (~> 1.6)
+  pundit-matchers (~> 1.7)
```

Comment:
looks like dependabot changed the pundit matchers version without checking in the gem. Fixing.